projectile.el

```diff
@@ -317,6 +317,7 @@ The topmost match has precedence."
     ".git"        ; Git VCS root dir
     ".hg"         ; Mercurial VCS root dir
     ".fslckout"   ; Fossil VCS root dir
+    "_FOSSIL_"    ; Fossil VCS root DB on Windows
```

Comment:
I'll go back through the history and find those bugfix commits to make sure; This may be a case for another `when` as below. What I do know is that `_FOSSIL_` is the traditional name for the checkout DB and the default on Windows today. The Fossil docs note that the convention may change in the future, but as of now the latest progress in that direction is a change from August 2012 that made `.fslckout` the default on Unix only (See version 1.23 in the changelog).

Something I didn't realize until now was that `_FOSSIL_` can be safely renamed to `.fslckout` even on Windows, giving Windows users an alternate workaround. This seems inelegant to me, however, and I'd prefer to get some form of this change merged, maybe with special logic for Windows systems.

https://www.fossil-scm.org/xfer/doc/trunk/www/tech_overview.wiki
https://www.fossil-scm.org/index.html/doc/trunk/www/changes.wiki